CONTRIBUTING.md

```diff
@@ -1,25 +1,14 @@
 # Introduction
 
-Azure Device OS Configuration (OSConfig) is a modular services stack running on a Linux IoT Edge device that facilitates remote device management over Azure as well from local management authorities. OSConfig contains a PnP Agent, a Management Platform (Modules Manager) and several Management Modules. For more information on OSConfig see [OSConfig North Star Architecture](docs/architecture.md) and [OSConfig Roadmap](docs/roadmap.md).
+Azure Device OS Configuration (OSConfig) is a modular configuration stack for�Linux Edge devices. OSConfig supports multi-authority device management over Azure and Azure Portal/CLI (via Azure PnP, IoT Hub, Azure Policy), GitOps, as well as local management (such as from OOBE). For more information on OSConfig see [OSConfig North Star Architecture](docs/architecture.md).
```

Comment:
I would recommend use Out of Box Experience (OOBE) instead of just OOBE

Reply:
Agree. Please review the update.